src/components/LineChart.tsx

```diff
@@ -61,25 +67,41 @@ function LineChart({
       className={`${className} ${
         rotate
           ? "rotate-90 h-[390px] w-[90vh] bg-inherit ml-[-50vw] mt-[22vh]"
-          : "relative ml-[5vw] sm:ml-[1vw] mr-[5vw] sm:mr-[0] md:mr-[0] sm:h-[340px] bg-[#22262C]"
+          : "relative ml-[5vw] sm:ml-[1rem] mr-[5vw] sm:mr-[0] md:mr-[0] sm:h-[340px] bg-[#22262C]"
```

Comment:
md:mr-[0] can be removed as sm and md values are same

Reply:
done